JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -34,8 +34,8 @@ pHMeter,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device that measures t
 Spectrometer,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device used to measure and analyze the spectrum of light.,https://www.theworldavatar.com/kg/ontodevice/,Spectrometer
 InfraredSpectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that analyzes infrared light spectra.,https://www.theworldavatar.com/kg/ontodevice/,Infrared spectrometer
 MassSpectrometer,Class,Spectrometer,IS-A,,,,A device used to measure the mass-to-charge ratio of ions.,https://www.theworldavatar.com/kg/ontodevice/,Mass spectrometer
-UltravioletVisibleSpectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that measures ultraviolet and visible light.,https://www.theworldavatar.com/kg/ontodevice/,Ultraviolet-visible spectrometer
-Xray_Spectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that analyzes X-ray spectra.,https://www.theworldavatar.com/kg/ontodevice/,X-ray spectrometer
+UltravioletVisibleSpectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that measures ultraviolet and visible light.,https://www.theworldavatar.com/kg/ontodevice/,Ultraviolet visible spectrometer
+Xray_Spectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that analyzes X-ray spectra.,https://www.theworldavatar.com/kg/ontodevice/,X ray spectrometer
```

Comment:
Sorry, I wasn't referring to the comments. You missed the only important occurrence, namely the `Xray_Spectrometer` class name. Please remove that underscore.

Also, please revert the changes in the comments you made - I do think all the hyphens were/are correct.